### PR TITLE
Add `mysqlclient` support

### PR DESCRIPTION
**Pull request by bitterpanda63**

### CODE REVIEW

File: sample-apps/django-mysql/manage.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-import aikido_firewall # Aikido module
-aikido_firewall.protect("django")
-
 import os
 import sys
 
 
 def main():
     """Run administrative tasks."""
+    import aikido_firewall # Aikido module
```
Comment (willem-delbare) [inline sample-apps/django-mysql/manage.py]: van boven

File: aikido_firewall/sinks/mysqlclient.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    def aikido_new_query(_self, sql):
+        logger.debug("Wrapper - `mysqlclient` version : %s", version("mysqlclient"))
+
+        context = get_current_context()
+        result = check_context_for_sql_injection(sql, "Test_op", context, MySQL())
+        result = check_context_for_sql_injection(
+            sql.decode("utf-8"), "Test_op", context, MySQL()
+        )
+
+        logger.info("sql_injection results : %s", json.dumps(result))
```
Comment (willem-delbare) [inline aikido_firewall/sinks/mysqlclient.py]: debug?

File: aikido_firewall/sinks/mysqlclient.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    """
+    modified_mysql = importhook.copy_module(mysql)
+
+    prev_query_function = copy.deepcopy(mysql.Connection.query)
+
+    def aikido_new_query(_self, sql):
+        logger.debug("Wrapper - `mysqlclient` version : %s", version("mysqlclient"))
+
+        context = get_current_context()
+        result = check_context_for_sql_injection(sql, "Test_op", context, MySQL())
```
Comment (willem-delbare) [inline aikido_firewall/sinks/mysqlclient.py]: test_op?

File: aikido_firewall/sinks/mysqlclient.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    modified_mysql = importhook.copy_module(mysql)
+
+    prev_query_function = copy.deepcopy(mysql.Connection.query)
+
+    def aikido_new_query(_self, sql):
+        logger.debug("Wrapper - `mysqlclient` version : %s", version("mysqlclient"))
+
+        context = get_current_context()
+        result = check_context_for_sql_injection(
+            sql.decode("utf-8"), "Test_op", context, MySQL()
```
Comment (willem-delbare) [inline aikido_firewall/sinks/mysqlclient.py]: test_op

File: aikido_firewall/sinks/mysqlclient.py (hunk 3 above)
Comment (bitterpanda63) [reply aikido_firewall/sinks/mysqlclient.py]: Done

**Pull request merged**